OpenXmlPowerTools/HtmlToWmlConverterCore.cs

```diff
@@ -2829,9 +2829,9 @@ private static XElement[] GetSpacingProperties(XElement paragraph, HtmlToWmlConv
             // todo should check based on display property
             bool numItem = paragraph.Name == XhtmlNoNamespace.li;
 
-            if (numItem && marginTopProperty.IsAuto)
+            if (numItem && marginTopProperty != null && marginTopProperty.IsAuto)
                 beforeAutospacing = "1";
-            if (numItem && marginBottomProperty.IsAuto)
+            if (numItem && marginTopProperty != null && marginBottomProperty.IsAuto)
```

Comment:
I think this should be `marginBottomProperty != null` (copy/paste issue I guess)

Reply:
if (numItem && marginBottomProperty?.IsAuto == true)